### DISCUSSION TITLE
How to create synonymies in the Taxonomy module.

### DISCUSSION THREAD

**Discussion by fmjjones** [fieldmuseum/EMu-Documentation#498] [HOW TO...]
How do I create synonymies in the Taxonomy module?

[CURRENT Google doc workflow.](https://docs.google.com/document/d/1PnftDxe-oRp1VFrlQsoFSFx796S4xKzPJxdLJDtdSv0/edit?usp=sharing)

**Comment by rondlg** [comment]
[DEPRECATED 2006 document - TaxFunction2006.pdf](https://github.com/fieldmuseum/EMu-Documentation/files/8483363/TaxFunction2006.pdf)
